mtp_noms_ops/apps/security/views/check.py

```diff
@@ -47,6 +52,32 @@ class CreditsHistoryListView(SecurityView):
     form_class = CreditsHistoryListForm
 
 
+class AutoAcceptRuleListView(SecurityView):
+    """
+    View history of all auto-accept rules
+    """
+    title = gettext_lazy('Auto Accepts')
+    template_name = 'security/auto_accept_rule_list.html'
+    form_class = AutoAcceptListForm
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['request_params'] = self.request.GET.dict()
+        return context
+
+
+class AutoAcceptRuleDetailView(SecurityDetailView):
```

Comment:
am i right in understanding that this view is incomplete? we should not have it in the url routes nor show the "review" button being visible at this stage. otherwise users will see a non-functioning button
```suggestion
class AutoAcceptRuleDetailView(SecurityDetailView):
```
```suggestion
class AutoAcceptRuleDetailView(SecurityDetailView):
```

Reply:
no idea what these "Suggested change" boxes are… i didn't put them there :)

Reply:
Yep, this is incomplete, I only added it because I needed the url to add to the templating for the review button, and that URL had to point to a view. Happy to comment this all out till the next ticket